components/brave_rewards/browser/service_sandbox_type.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_REWARDS_BROWSER_SERVICE_SANDBOX_TYPE_H_
+#define BRAVE_COMPONENTS_BRAVE_REWARDS_BROWSER_SERVICE_SANDBOX_TYPE_H_
+
+#include "content/public/browser/service_process_host.h"
+
+namespace bat_ledger {
```

Comment:
`#if !defined(OS_ANDROID)  // Android will use default, which is kUtility.`

Why do we drop this? Do we not need it?

Reply:
Oops, it's my mistake. Added again.